fivetran_provider/operators/fivetran.py

```diff
@@ -81,3 +83,65 @@ def execute(self, context):
         hook = self._get_hook()
         hook.prep_connector(self.connector_id, self.schedule_type)
         return hook.start_fivetran_sync(self.connector_id)
+
+    def _get_fields(self, table) -> Optional[Field]:
+        if table.get("columns"):
+            return [
+                Field(
+                    name=col["name_in_destination"],
+                    type="",
+                )
+                for col in table["columns"].values()
+            ]
+        return None
```

Comment:
This should return `SqlJobFacet`

Reply:
Not a `SchemaField`? I want to return column information here.